src/Reflection/ReflectionFunction.php

```diff
@@ -38,10 +41,13 @@ public static function createFromClosure(Closure $closure) : self
     {
         $configuration = new BetterReflection();
 
-        return (new FunctionReflector(
+        /** @var self $reflectionFunction */
```

Comment:
I think you can revert this change and instead update the docblock of `FunctionReflector::reflect` instead to reflect that it always returns a `ReflectionFunction`. PHPStan [allows that syntax](https://phpstan.org/r/43781f0e8344ccf014130042a3a232af).

Reply:
You,re true, thanks.